packages/saltcorn-data/models/table.ts

```diff
@@ -152,24 +152,29 @@ class Table implements AbstractTable {
   /**
    *
    * Find one Table
+   *
    * @param where - where condition
    * @returns {*|Table|null} table or null
    */
-  static findOne(where: Where | Table): Table | null {
+  static findOne(where: Where | Table ): Table | null {
     if (
       where &&
       ((where.constructor && where.constructor.name === "Table") ||
         where.getRows)
     )
       return <Table>where;
+    // todo remove this logic. But needs find all usages! Also this is recursion that usually adds call function overhead
```

Comment:
I don't think they should be removed because it is used by plug-ins to find their associated table. It's OK to reduce the usage though